package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "express-generator",
   "description": "Express' application generator",
-  "version": "4.16.1",
+  "version": "5.0.0",
```

Comment:
The generator version matches the generated express version. Make sure to update the related version of Express as well, and you can remove this line from the PR, as npm command would then cause us to release as a diff versions since it bumps what is in the package when making the tag and publishing.